### DISCUSSION TITLE
This chip is ESP32 not ESP8266. Wrong --chip argument?

### DISCUSSION THREAD

**Discussion by meilleur102** [Bodmer/TFT_eSPI#2525] [Q&A - General]
Hi!. I'm trying to make a ESP32 work with this library and I can't figure it out. 

Ive try with a Wemos ESP8266 D1 mini and I was not able to get any image on the screen 
(This is the display http://www.lcdwiki.com/3.5inch_SPI_Module_ILI9488_SKU:MSP3520).
I had a Esp32 around and I try to make this one work to see if it's a board issue, but im not able to upload the program. 
I have this error when I try to upload. 
This chip is ESP32 not ESP8266. Wrong --chip argument?

Thanks
Jc

**Comment by meilleur102** [comment]
Got it. Was changing the UserSetup at the wrong place like a noob. Sorry

**Reply by manulsan** [reply]
I have same problem and any progress?
